src/components/map/tools/EditNetworkNodeTool.ts

```diff
@@ -1,34 +0,0 @@
-import NetworkStore, { NodeSize } from '~/stores/networkStore';
```

Comment:
Maybe we could re add this?
We currently do not use it, but might well we need it if we want to allow opening a node/link by clicking on map.